### PR TITLE
Relax activerecord version

### PR DESCRIPTION
**Pull request by sauloperez**
dependencies versions in gems should be flexible enough to allow bundler fulfil all the versions specified in an app's Gemfile.

The `<= 3.2.19` constraint for activerecord made impossible to update to the latest rails security update that comes with activerecord `3.2.22.1`

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,43 +0,0 @@
-PATH
```
Comment (jhbabon) [inline Gemfile.lock]: In a gem, this file needs to be ignored as well.

**Comment by jhbabon** [comment]
Apart of my comment, :+1:

File: Gemfile.lock (hunk 1 above)
Comment (sauloperez) [inline Gemfile.lock]: true

**Comment by franciscoj** [comment]
:+1:

**Pull request merged**